hutch_python/epics_arch.py

```diff
@@ -156,33 +156,93 @@ def create_arch_file(experiment, level=None, hutch=None, path=None, dry_run=Fals
         print_dry_run(experiment)
 
 
-def pull_cds_data(exp, run):
+def pull_cds_items(exp):
+    logger.debug("in client")
+
     """
-    Gather all user objects from the CDS tab in the questionnaire.
-    Parse objects and separate them based on type.
-    Display them in the console via PrettyTable.
+    Gather all user obejcts from the CDS tab in the questionnaire.
```

Comment:
Did you edit these an intentionally mispell them?...

Reply:
I'm assuming this is a git mishap from a previous PR where we fixed typos as in-line suggestions in github but they didn't propagate to his fork